src/main/java/uk/gov/ida/verifyserviceprovider/configuration/HubEnvironment.java

```diff
@@ -20,20 +32,30 @@
 
 public enum HubEnvironment {
     PRODUCTION(
-            URI.create("https://www.signin.service.gov.uk/SAML2/SSO"),
-            URI.create("https://www.signin.service.gov.uk/SAML2/metadata/federation"),
+            URI.create(PRODUCTION_SSO),
+            URI.create(PRODUCTION_METADATA),
+            URI.create(PRODUCTION_METADATASOURCE_URI),
+            URI.create(PRODUCTION_TRUSTANCHOR_URI),
             PRODUCTION_METADATA_TRUSTSTORE, PRODUCTION_HUB_TRUSTSTORE, PRODUCTION_IDP_TRUSTSTORE),
     INTEGRATION(
-            URI.create("https://www.integration.signin.service.gov.uk/SAML2/SSO"),
-            URI.create("https://www.integration.signin.service.gov.uk/SAML2/metadata/federation"),
+            URI.create(INTEGRATION_SSO),
+            URI.create(INTEGRATION_METADATA),
+            URI.create(INTEGRATION_METADATASOURCE_URI),
+            URI.create(INTEGRATION_TRUSTANCHOR_URI),
             TEST_METADATA_TRUSTSTORE, TEST_HUB_TRUSTSTORE, TEST_IDP_TRUSTSTORE),
     COMPLIANCE_TOOL(
-            URI.create("https://compliance-tool-reference.ida.digital.cabinet-office.gov.uk/SAML2/SSO"),
-            URI.create("https://compliance-tool-reference.ida.digital.cabinet-office.gov.uk/SAML2/metadata/federation"),
+            URI.create(COMPLAINCE_SSO),
+            URI.create(COMPLAINCE_METADATA),
+            URI.create(COMPLIANCE_METADATASOURCE_URI),
+            URI.create(COMPLIANCE_TRUSTANCHOR_URI),
             TEST_METADATA_TRUSTSTORE, TEST_HUB_TRUSTSTORE, TEST_IDP_TRUSTSTORE);
 
+
     private URI ssoLocation;
     private URI metadataUri;
+
+    private URI eidasmetaDataSourceUri;
```

Comment:
Sorry to be picky, but this should be `eidasMetadataSourceUri` note capitalisation.

Reply:
Thanks for this